

syntax = "proto3";


service Greeter {
  rpc SayHello(HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string data = 1;
  int32 status_code = 2;
}

message HelloReply {
  string result = 1;
  int32 status_code = 2;
}

